.dialyzer_ignore.exs

```diff
@@ -10,5 +10,9 @@
   {"lib/atecc508a/certificate.ex:37:unknown_type Unknown type: :public_key.ec_private_key/0."},
   {"lib/atecc508a/certificate/template.ex:9:unknown_type Unknown type: :public_key.ec_public_key/0."},
   {"lib/atecc508a/certificate/template.ex:9:unknown_type Unknown type: :public_key.rsa_public_key/0."},
-  {" lib/atecc508a/certificate/template.ex:10:unknown_type Unknown type: X509.ASN1.record/1."}
+  {" lib/atecc508a/certificate/template.ex:10:unknown_type Unknown type: X509.ASN1.record/1."},
```

Comment:
```suggestion
  {"lib/atecc508a/certificate/template.ex:10:unknown_type Unknown type: X509.ASN1.record/1."},
```

I know that space was originally there, but it just looks wrong... 

Speaking of which, shouldn't all of the `template.ex` lines go away since the file was renamed to `nerves_key_template.ex`?